docs/concepts/storage/persistent-volumes.md

```diff
@@ -220,6 +220,12 @@ Current reclaim policies are:
 
 Currently, only NFS and HostPath support recycling. AWS EBS, GCE PD, Azure Disk, and Cinder volumes support deletion.
 
+When the Retain reclaim policy is specified, you can manually reclaim your volume with the following steps.
```

Comment:
I just noticed there's a section at the top about the PVC/PV lifecycle.  Maybe this would be better up there.